infra/src/main/resources/application.yml

```diff
@@ -108,7 +108,7 @@ geoviite:
       enabled: true
```

Comment:
Onkohan meillä joku syy minkä takia ProjektiVelhon pitää ylipäätään olla enabled E2E-testien ajossa? Mietin voisikohan sen vaan disabloida kokonaan niissä 🤔 Kyseessähän ei ole Ratko-integraation tyyppinen juttu jossa meillä enabloituu/disabloituu juttuja UI:ltakin sen perusteella onko integraatio päällä vai ei, vaan se vaikuttaa oikeastaan pelkästään näiden poll-taskien ajamiseen

Reply:
Meillä ei ole erillistä Spring-profiilia E2E-testeille, testi-infra jota käytetään E2E-testejä varten käyttää suoraan test-profiilia (voitaisiin toki tehdä erillinen profiili tai hankkiutua testi-infrasta eroon kokonaan).

Tuossa test-profiilissa projektivelhon enabled=true flagiä taas tarvitaan IT-testien ajoon (ks. PVIntegrationServiceIT.kt)

Reply:
Hups, olinkin väärässä, koska e2e-profiili ei olekaan test-kansion alla (application-e2e). Kaipa tuon voisi sieltä kytkeä pois päältäkin oikeastaan.

Reply:
Profiileja ja propsuja on myös mahdollista yliajaa siellä testiluokan annotaatioissa jos on kyse sellaisesta featuresta jota kaipaa tasan yksi testitiedosto